### PR TITLE
Add Linux cluster builder

### PR DESCRIPTION
**Pull request by skw1992**

### CODE REVIEW

File: deploy/deployGearpump.sh [hunk 1]
```diff
@@ -0,0 +1,132 @@
+#!/bin/bash 
+alias ssh="ssh -i ~/.ssh/id_rsa";
+alias scp="scp -i ~/.ssh/id_rsa";
+gearpumpVersion="pack-2.11.5-0.6.2-SNAPSHOT"
```
Comment (whjiang) [inline deploy/deployGearpump.sh]: it's bad to have version info hard coded.

File: deploy/deployGearpump.sh [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+gearpumpDir="/root/gearpump-${gearpumpVersion}"
+gearpumpPack="/root/gearpump-${gearpumpVersion}.tar.gz" 
+
+#cluster="intelidh-01 intelidh-03 intelidh-04 intelidh-06"
+#cluster="node6 node7 node8 node10 node11 node12 node16 node17 node18 node19 node20 node21"
+#cluster="node6 node12 node16 node17 node19"
+#cluster="node6 node12 node16 node17 node19 node20"
+#cluster="node18 node17 node19 node20"
+cluster="IDHV22-01 IDHV22-06"
```
Comment (whjiang) [inline deploy/deployGearpump.sh]: it's bad to have cluster info hard coded.

File: linux-test/src/main/scala/org/apache/gearpump/test/linux/Util.scala [hunk 3]
```diff
@@ -22,20 +22,13 @@ object Util extends App {
   lazy val httpsProxy = "https_proxy" -> "http://child-prc.intel.com:913"
   lazy val address = "https://github.com/intel-hadoop/gearpump.git"
   lazy val build = "sbt clean assembly packArchive"
-  lazy val sourceRoot = "/root/gearpump-yarn-test/gearpump"
+  lazy val sourceRoot = "/root/gearpump-test/gearpump"
```
Comment (stanleyxu2005) [inline linux-test/src/main/scala/org/apache/gearpump/test/linux/Util.scala]: `/root/gearpump-test` means to run tests under root account. Better create a random temporary directory for testing.

File: linux-test/src/main/scala/org/apache/gearpump/test/linux/LinuxClusterBuilder.scala [hunk 4]
```diff
@@ -0,0 +1,41 @@
+package org.apache.gearpump.test.linux
+
+import java.io.File
+
+import scala.sys.process.Process
+
+object LinuxClusterBuilder extends Builder{
+
+  lazy val sourceRoot = "/root/gearpump-test/gearpump"
```
Comment (stanleyxu2005) [inline linux-test/src/main/scala/org/apache/gearpump/test/linux/LinuxClusterBuilder.scala]: The ultimate root directory should be temporary directory.

File: linux-test/src/main/scala/org/apache/gearpump/test/linux/LinuxClusterBuilder.scala [hunk 5]
```diff
@@ -0,0 +1,58 @@
+package org.apache.gearpump.test.linux
```
Comment (manuzhang) [inline linux-test/src/main/scala/org/apache/gearpump/test/linux/LinuxClusterBuilder.scala]: we use `io.gearpump` for package name now

**Comment by stanleyxu2005** [comment]
+1

**Pull request merged**